test/e2e/snaps/test-snap-cronjob.spec.js

```diff
@@ -74,6 +75,8 @@ describe('Test Snap Cronjob', function () {
           text: 'Reconnect to Cronjobs Snap',
         });
 
+        await driver.delay(largeDelayMs);
+
         // switch to dialog popup, wait for a maximum of 65 seconds
```

Comment:
We should remove the comment `wait for a maximum of 65 seconds`

It used to do that in a previous version, but it no longer does, and that's actually the problem here.  It used to have
`windowHandles = await driver.waitUntilXWindowHandles(3, 1000, 65000);`


I researched it, and there's no easy way to increase the timeout in my new `switchToWindowWithTitle` (oops)

Reply:
Sure, I will remove the comment. Thanks for checking @HowardBraham.